api_types.ts

```diff
@@ -560,8 +560,17 @@ export interface Logger {
   error(message: string, ...args: LoggerParamType[]): void;
 }
 
+/**
+ * Information about the Coda environment and doc this formula was invoked from.
+ * This is mostly for Coda internal use and we do not recommend relying on it.
```

Comment:
I'm fine saying we're not going to support it. Let's operate based on that and we can re-introduce it if legitimate use cases arise.

We can just rip it out immediately because it looks like the Coda Admin pack relies on it, so we would need to introduce an alternative (maybe only setting it for specific known internal packs). It's already optional, I've marked it as deprecated and said it's going away.

Reply:
*CAN'T just rip it out immediately